spec/controllers/spells_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe SpellsController, type: :controller do
+
```

Comment:
If we are going to include the spec we might as well request the page and assert the response is 200

Reply:
I forgot about this file. Removed.